std/didexchange/v0/request_impl.go

```diff
@@ -109,6 +109,10 @@ func (m *requestImpl) VerKey() string {
 }
 
 func (m *requestImpl) Endpoint() service.Addr {
+	defer err2.Catch(err2.Err(func(err error) {
```

Comment:
**Just** FYI: with the current err2 allows this catch-block shortened, and `cmd/root.go` hast `glog.CopyStandardLogTo("ERROR")` which gives pretty much the same result:
```go
defer err2.Catch()
```
Why this is little better, it's the refactoring and copy-pasting, not the amount of lines some much.